### DISCUSSION TITLE
List Icons

### DISCUSSION THREAD

**Discussion by QaahirStewart** [unocss/unocss#1733] [Q&A]
Is there a way to list Icons or fetch icons json data? For ex. I want to create a dropdown that shows icons that are pickable for each iconset MDI, Font Awsome etc.

**Comment by SnowingFox** [comment]
hi, i think you can refer [this](https://github.com/unocss/unocss/tree/main/packages/preset-icons#bundler)

**Comment by zyyv** [comment]
Hi, i create a PR to download the iconset json data. https://github.com/antfu/icones/pull/96
I think the best way is to use the icon you want to use locally, so i create an issue. https://github.com/antfu/icones/issues/97 (: This coincides with on-demand😅

Well, I think the whole iconset data is too huge, if you want fetch the iconset json data, you can refer to https://github.com/antfu/icones/blob/main/scripts/prepare.ts
